File: Server/src/main/content/global/skill/production/cooking/recipe/potato/impl/CheesePotato.java
package content.global.skill.production.cooking.recipe.potato.impl;

import content.global.skill.production.cooking.recipe.potato.PotatoRecipe;
import core.game.node.item.Item;

/**
 * The Cheese potato.
 */
public class CheesePotato extends PotatoRecipe {

    private static final Item CHEESE_POTATO = new Item(6705);

    private static final Item CHEESE = new Item(1985);

    @Override
    public Item getProduct() {
        return CHEESE_POTATO;
    }

    @Override
    public Item[] getIngredients() {
        return new Item[]{CHEESE};
    }

    @Override
    public boolean isTopping() {
        return false;
    }

    @Override
    public int getLevel() {
        return 47;
    }

    @Override
    public double getExperience() {
        return 10;
    }

}

File: Server/src/main/content/global/skill/production/cooking/recipe/potato/impl/CheesePotato.kt
package content.global.skill.production.cooking.recipe.potato.impl

import content.global.skill.production.cooking.recipe.potato.PotatoRecipe
import core.api.consts.Items
import core.game.node.item.Item

/**
 * Cheese potato.
 */
class CheesePotato : PotatoRecipe() {

    override fun getProduct(): Item {
        return CHEESE_POTATO
    }

    override fun getIngredients(): Array<Item> {
        return arrayOf(CHEESE)
    }

    override fun isTopping(): Boolean {
        return false
    }

    override fun getLevel(): Int {
        return 47
    }

    override fun getExperience(): Double {
        return 10.0
    }

    companion object {
        private val CHEESE_POTATO = Item(Items.POTATO_WITH_CHEESE_6705)
        private val CHEESE = Item(Items.CHEESE_1985)
    }
}
